tests/models/test_schema.py

```diff
@@ -0,0 +1,161 @@
+        obj.pop("sameAs", None)  # not in models
+        obj.pop("subPropertyOf", None)  # not in models
+        obj.pop("description", None)  # not in spec
+        obj.pop("default", None)  # not in spec
+        if obj.get("type") == "null":  # pydantic 2 sets these
+            obj.pop("type")
+        if obj.get("examples") == [str(Identifier.generate(seed=42))]:  # not in spec
+            obj.pop("examples")
+        if field_name == "temporal":  # this field's examples are too complicated
+            obj.pop("examples", None)
+        if obj.get("format") == "date":  # we mash up dates and datetimes in models
+            obj["format"] = "date-time"
+        if obj.get("pattern") in (
+            r"^[1-9]\d{3}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$",
+            r"^\d{4}(-\d{2})?(-\d{2})?$",  # ignore date/time pattern differences
+        ):
+            obj["pattern"] = r"^\d{4}(-\d{2}(-\d{2}(T\d{2}:\d{2}:\d{2}Z)?)?)?$"
+        if obj.get("pattern") == MEX_ID_PATTERN:  # align concept/enum annotations
+            obj.pop("pattern")
+            obj.pop("type")
+            if obj.get("title") and field_name not in ("identifier", "stableTargetId"):
+                title = dromedary_to_kebab(obj.pop("title").removesuffix("ID"))
+                obj["$ref"] = f"/schema/entities/{title}#/identifier"
+            else:
+                obj["$ref"] = "/schema/fields/identifier"
+                obj.pop("title", None)
+        else:
+            obj.pop("title", None)
+        if "$ref" in obj:  # align reference paths
+            obj["$ref"] = obj["$ref"].replace("#/$defs/", "/schema/fields/").lower()
+            if obj["$ref"] == "/schema/entities/concept#/identifier" and (
+                scheme := obj.pop("useScheme", None)
+            ):
+                name = scheme.replace("-", "").removeprefix("https://mex.rki.de/item/")
+                obj["$ref"] = f"/schema/fields/{name}"
+        if obj.get("type") == "array":
+            obj.pop("default", None)  # ignore empty list defaults
+            if isinstance(obj["items"], dict):
+                obj["items"] = [obj["items"]]
+            if scheme := obj.pop("useScheme", None):  # pull down scheme into items
+                for item in obj["items"]:
+                    item["useScheme"] = scheme
+            prepare_field(obj["items"])
+            if any(
+                item.get("examples") if isinstance(item, dict) else None
+                for item in obj["items"]
+            ):  # pull up examples onto field level
+                obj["examples"] = [
+                    e
+                    for item in obj["items"]
+                    for e in (
+                        item.pop("examples", []) if isinstance(item, dict) else []
+                    )
+                ]
+        if "anyOf" in obj:  # ignore any/one difference
+            obj["oneOf"] = obj.pop("anyOf")
+        for quantifier in ("oneOf", "allOf"):
+            if quantifier in obj:
```

Comment:
Not sure if this is easy enough to read but at least on less level of indentation 😝 

```suggestion
        for quantifier in ({"oneOf", "allOf"} & set(obj)):
```